client/finality-grandpa/src/finality_proof.rs

```diff
@@ -132,7 +132,7 @@ pub struct FinalityProof<Header: HeaderT> {
 	/// The hash of block F for which justification is provided.
 	pub block: Header::Hash,
 	/// Justification of the block F.
-	pub justification: Vec<u8>,
+	pub justification: EncodedJustification,
```

Comment:
I know that type wrappers are cool, but I don't understand what this brings? Someone who reads the code now can see that it's a `Vec<u8>`, whereas in the future they will have to go look inside `sp_runtime` to see what `EncodedJustification` is. Apart from increasing reading difficulty, this type seems to bring nothing?

```suggestion
	pub justification: Vec<u8>,
```

Reply:
I think in this case it was added to emphasize that it's a single encoded `Justification`, and not an encoded `Justifications`. But then again in this context here it's debatable how useful that is ....